products.d/leap_micro_62.yaml

```diff
@@ -0,0 +1,110 @@
+id: LeapMicro_6.2
+name: openSUSE Leap Micro 6.2 Beta
```

Comment:
You might want to add the archs list here: `archs: x86_64,aarch64`.

Reply:
Yes!